docs/docsite/requirements.in

```diff
@@ -0,0 +1,9 @@
+# This requirements file is used for AWX latest doc builds.
+
+-c constraints.in # <-- contains known limitations
```

Comment:
Refer to https://github.com/ansible/awx/pull/14449/files#r1328689563 to decide if it's really needed.